api/controllers/users.py

```diff
@@ -532,6 +532,7 @@ def model_upload_s3_dynalab_2(credentials):
                 "user_id": user_id,
                 "user_name": user_name,
                 "task_code": task_code,
+                "model_id": model_id,
```

Comment:
thanks that's good progress. 
Could we also have the "model_secret" ? This is required to call `api.dynabench.org/models/{model_id}/download`

https://github.com/mlcommons/dynabench/blob/0d0b9f68ea4cc56e0b8799b60e2db4a66bd910fd/builder/utils/helpers.py#L67-L67

Reply:
Done